<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
	color: black;
}

QGroupBox{
	/*border: none;*/
	color: black;
}

QPushButton{
	background-color:#486161;
	border-radius: 10px;
	color: #f8f8f8;
	font-weight: bold;
}

QPushButton:hover{
	background-color: #5d8282;
	border: 2px solid #91c3ad;
	color: #f8f8f8;
	font-weight: bold;
}

QLineEdit, QDateEdit {
	border: 1px solid #91c3ad;
	border-radius: 7px;
	background-color: #deeee1;
	color: black;
}</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>271</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Usuario :</string>
      </property>
      <widget class="QLineEdit" name="user_txt">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>250</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Contrase√±a :</string>
      </property>
      <widget class="QLineEdit" name="password_txt">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>250</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>241</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="save_btn">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="return_btn">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Volver</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="title_lbl">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Registrar usuario</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
